src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexParser.cs

```diff
@@ -1052,13 +1050,13 @@ private void ScanBlank()
                 if ((_options & RegexOptions.IgnorePatternWhitespace) != 0 && _pos < _pattern.Length && _pattern[_pos] == '#')
                 {
                     _pos = _pattern.IndexOf('\n', _pos);
-                    if (_pos == -1)
+                    if (_pos < 0)
                         _pos = _pattern.Length;
```

Comment:
braces

Reply:
ah, I wasn't aware of the coding guidelines that braces are required in a nested context